davidagitlen/Tripedia-fe
# Refactor form hooks

#### What’s this PR do?
- Refactor Hooks in login and createAccount to be a single state object.

#### Where should the reviewer start?
- At the top of Login and CreateAccount components you will notice that there is one hook with a handler function. 
#### How should this be manually tested?
- npm start and go to the '/login' and 'create_account' endpoints, open React devtools and notice that when you type in an input the state reflects the changes on the dom.
